content/articles/user-input-and-while-loops-in-python/index.md

```diff
@@ -0,0 +1,198 @@
+As a programmer, many programs that you'll write will solve an end user's problem. You will always want to get some data from the user. This article will teach us how to accept user input and how to use while loops. While loops enable your programs to remain executing when certain conditions remain true.
+
+### Prerequisites
+1. Python installed on your computer.
+2. Basic Python knowledge.
+
+### How the input() function works
+The `input()` function halts your program and waits the user to key in some data. When Python receives the user's input, store it in the variable that you choose to work with.
+
+For example, let's create a program that accepts a user's name and prints back the name.
+
+```python
+name = input("Please enter your name: ")
+print(name)
+#output
+#Please enter your name: John
+#John
+```
+
+The `input()` function requires one argument, that is, the instruction you want the user to see. In this example, Python executes the first line and requests the user to enter his name. The program will halt and wait for the user to enter his name and continue after he presses **ENTER**. The name of the user is then loaded in the variable *name*, then `print(name)` displays the user's name back.
+
+### Using int() to accept numerical input
+Any text the user enters using the `input()` function Python interprets it as a string. If your work is to print the input, the `input()` function works very well. But if you use the input as a number, an error arises. Consider the example below:
```

Comment:
Rewrite "Any text the user enters using the `input()`..." to "Any text the user input using the `input()`..."